azure-pipelines.yml

```diff
@@ -160,33 +164,3 @@ extends:
                   name: Build
                   displayName: Build
                   condition: succeeded()
-
-    - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-      - template: eng/common/templates-official/post-build/post-build.yml@self
-        parameters:
-          publishingInfraVersion: 3
-          # Symbol validation isn't being very reliable lately. This should be enabled back
-          # once this issue is resolved: https://github.com/dotnet/arcade/issues/2871
-          enableSymbolValidation: false
-          # Sourcelink validation isn't passing for Arcade due to some regressions. This should be
-          # enabled back once this issue is resolved: https://github.com/dotnet/arcade/issues/2912
-          enableSourceLinkValidation: false
-          publishDependsOn:
-          - Validate
-          # This is to enable SDL runs part of Post-Build Validation Stage
-          SDLValidationParameters:
-            enable: true
-            continueOnError: false
-            params: ' -SourceToolsList @("policheck","credscan")
-            -ArtifactToolsList @("binskim")
-            -BinskimAdditionalRunConfigParams @("IgnorePdbLoadError < True","Recurse < True")
-            -TsaInstanceURL $(_TsaInstanceURL)
-            -TsaProjectName $(_TsaProjectName)
-            -TsaNotificationEmail $(_TsaNotificationEmail)
-            -TsaCodebaseAdmin $(_TsaCodebaseAdmin)
-            -TsaBugAreaPath $(_TsaBugAreaPath)
-            -TsaIterationPath $(_TsaIterationPath)
-            -TsaRepositoryName $(_TsaRepsitoryName)
-            -TsaCodebaseName $(_TsaCodebaseName)
-            -TsaPublish $True'
```

Comment:
@marcpopMSFT since this part got removed dotnet/templating's internal builds don't publish to darc anymore and hence templating is stale in sdk. Any idea why this got removed?